diff --git a/node_modules/minimatch/dist/esm/index.js b/node_modules/minimatch/dist/esm/index.js
index 3175327..fcc9ed5 100644
--- a/node_modules/minimatch/dist/esm/index.js
+++ b/node_modules/minimatch/dist/esm/index.js
@@ -1019,4 +1019,5 @@ minimatch.AST = AST;
 minimatch.Minimatch = Minimatch;
 minimatch.escape = escape;
 minimatch.unescape = unescape;
+export default minimatch;
 //# sourceMappingURL=index.js.map
\ No newline at end of file
